### PR TITLE
[pixi.js] fix PIXI.js unable to fallback to Canvas

### PR DESCRIPTION
**Pull request by simonlingoogle**
This PR fixes the issue of PIXI.js unable to fallback to Canvas if WebGL is not supported.

### CODE REVIEW

File: web/site/js/vis/PixiVisualizer.js [hunk 1]
```diff
@@ -24,6 +24,28 @@
...
 
+var _isWebGLSupported = undefined
+
+function isWebGLSupported() {
+    if (typeof _isWebGLSupported !== "undefined") {
+        return _isWebGLSupported
+    }
+
+    try {
+
```
Comment (ctan-g) [inline web/site/js/vis/PixiVisualizer.js:35]: Remove blank line?

**Comment by ctan-g** [comment]
(Haven't tested on machines that doesn't have any WebGL support, I don't have one in handy)

**Comment by ctan-g** [comment]
Running chrome with `google-chrome --disable-3d-apis` gives the following warning (and draws nothing on the canvas):
Screenshot from 2020-07-31 15-36-18
Is the intended behavior to only warn user in the console if WebGL is not supported, instead of automatically falling back to `canvas2d`?

**Comment by simonlingoogle** [comment]
> Running chrome with `google-chrome --disable-3d-apis` gives the following warning (and draws nothing on the canvas):
> Screenshot from 2020-07-31 15-36-18
> Is the intended behavior to only warn user in the console if WebGL is not supported, instead of automatically falling back to `canvas2d`?

Great. I will try cavans2d. Might be more efficient than WebGL Canvas on some machine.

**Comment by simonlingoogle** [comment]
Hi @ctan-g , could you try latest PR with `google-chrome --disable-3d-apis` again? It should be working now. Thanks.

**Comment by jwhui** [comment]
@simonlingoogle , is this still work-in-progress?

**Comment by simonlingoogle** [comment]
> @simonlingoogle , is this still work-in-progress?

No, it's not . Thanks.

**Pull request merged**